pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -235,6 +235,21 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 				currentTimeoutMinutes++
 				if currentTimeoutMinutes == timeoutMinutes {
 					config.addError("timed out waiting for %d nodes: %+v", remainingNodes.Count(), remainingNodes)
+					//start to remove remainingNodes waiting timeout.
+					for remainingNode := range remainingNodes.Iter() {
+						if err := c.updateOSDStatus(
```

Comment:
I don't think we need to call `updateOSDStatus()` since that will just update the configmap immediately before it is deleted by `ClearStore()`.

Reply:
It makes sense. And I would delete it.